.github/CODEOWNERS

```diff
@@ -13,6 +13,7 @@
 
 # Angular Migration team
 /core/templates/dev/head/services/contextual/logger.service.ts @bansalnitish
+/core/templates/dev/head/services/contextual/logger.service.spec.ts @bansalnitish
```

Comment:
as you did below, you can probably just add a * at the end of the previous line?

Reply:
please take a look on kevin's comment above, he suggested to list all the files. Which pattern should I follow? /cc @kevinlee12

Reply:
Hmm, then shouldn't you do that below?

Reply:
this is fine for now.